packages/components/avatar/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  extends: ['custom/react'],
+  env: {
+    browser: true,
+    node: true,
+  },
+  rules: {
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/ban-ts-comment': 'off',
+    '@typescript-eslint/prefer-ts-expect-error': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-unsafe-member-access': 'off',
+    '@typescript-eslint/no-unsafe-assignment': 'off',
+    '@typescript-eslint/no-unsafe-call': 'off',
+    '@typescript-eslint/no-unsafe-return': 'off',
+    '@typescript-eslint/no-confusing-void-expression': 'off',
+    'react/no-array-index-key': 'off',
+    'react/no-unknown-property': 'off',
```

Comment:
Those rules are duplicate over other components. Should we move those rules to the base config?